nototools/generate_sample_text_html.py

```diff
@@ -62,18 +62,18 @@ def generate_table(filename):
           if bcp_en == 'Unknown Language' and sample_type == 'chars':
             bcp_en = '(characters)'
         except:
-          print 'could not get english name for %s' % bcp
+          print('could not get english name for %s' % bcp)
           bcp_en = bcp
 
         cols = ['<tr>']
         cols.append(bcp_no_script)
         cols.append(bcp_en)
         cols.append(sample_type)
         cols.append(sample_text)
-        print >> f, '<td>'.join(cols)
-      print >> f, '<tr><td colspan=4>&nbsp;'
-    print >> f, '</table>'
-    print >> f, _HTML_FOOTER
+        f.write('<td>'.join(cols) + '\n')
+      f.write('<tr><td colspan=4>&nbsp;')
```

Comment:
why no '\n' added in these lines?

Reply:
meant to make that a review comment

Reply:
Oh, you are right! adding these '\n' ...